### ISSUE TITLE
Design a success and failure page

### ISSUE THREAD

**Issue by sivaraam** [sirganeshv/Seminar#32]
* It must look neat.
* It must have some identification of success / failure
* In case of failure the reason must be clear
* Follow [Material design](http://material.io/) guidelines

First a mockup must be uploaded here.

**Comment by sivaraam** [comment]
@Aishulaksh Could you take a look into this ?

**Comment by Aishulaksh** [comment]
ya ok

**Comment by sivaraam** [comment]
@Aishulaksh How's this going?

**Comment by sivaraam** [comment]
Let me this for one last time,

@Aishulaksh **How's this going?**

**Comment by Aishulaksh** [comment]
Will let you know the status by today evening

**Comment by Aishulaksh** [comment]
I don't think it requires a new page for mentioning success or failure. It can be like an alert or some dialog box mentioning either success or failure and reason for the failure.

**Comment by sivaraam** [comment]
> I don't think it requires a new page for mentioning success or failure. It can be like an alert or some dialog box mentioning either success or failure and reason for the failure.

Not many would like the pop-ups (count me in). We won't lose much with showing a new page. Moreover, a whole page for success/failure gives more room than a dialog box and so would not restrict the space for design.

**Comment by Aishulaksh** [comment]
Then display in the centre of the new page. if success "Successfully booked" with a big thumbs up symbol. If suppose its an error, error icon with the error message displayed

**Comment by sivaraam** [comment]
> Then display in the centre of the new page. if success "Successfully booked" with a big thumbs up symbol. If suppose its an error, error icon with the error message displayed

I was asking you just that. Can you design it?

**Comment by Aishulaksh** [comment]
Oh i thought you're asking me idea! ya fine
